maquetado/index.html

```diff
@@ -10,5 +10,15 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="text-container">
```

Comment:
This class name it's not very descriptive, you change maybe for `techs-container` or other class name.